Database/Repositories/Interfaces/ITournamentsRepository.cs

```diff
@@ -194,4 +194,22 @@ Task<Dictionary<int, int>> GetLobbySizeStatsAsync(
     /// Gets a map of <see cref="Common.Enums.Verification.VerificationStatus"/>es to the number of <see cref="Database.Entities.Tournament"/>s with the status
     /// </summary>
     Task<Dictionary<VerificationStatus, int>> GetVerificationStatusStatsAsync();
+
+    /// <summary>
+    /// Gets a map of years to the number of <see cref="Database.Entities.Tournament"/>s in that year
+    /// </summary>
+    /// <remarks>Only verified tournaments are counted</remarks>
+    Task<Dictionary<int, int>> GetYearStatsAsync();
+
+    /// <summary>
+    /// Gets a map of <see cref="Common.Enums.Ruleset"/>s to the number of <see cref="Database.Entities.Tournament"/>s in that ruleset
+    /// </summary>
+    /// <remarks>Only verified tournaments are counted</remarks>
+    Task<Dictionary<Ruleset, int>> GetRulesetStatsAsync();
+
+    /// <summary>
+    /// Gets a map of lobby sizes to the number of <see cref="Database.Entities.Tournament"/>s with that lobby size
+    /// </summary>
+    /// <remarks>Only verified tournaments are counted</remarks>
+    Task<Dictionary<int, int>> GetLobbySizeStatsAsync();
```

Comment:
On second thought, I'd like to see a `bool verified = true` parameter for each of these rather than letting that be the only behavior.